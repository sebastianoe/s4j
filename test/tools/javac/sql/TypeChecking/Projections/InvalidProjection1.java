/*
 * @test
 * @compile/fail/ref=InvalidProjection1.out InvalidProjection1.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */
// key: compiler.err.sql.validation.types.not.compatible
import java.sql.ResultSet;

public class InvalidProjection1 {
	public static void main(String[] args) {
		int i = SQL[SELECT 3.14];
	}
}