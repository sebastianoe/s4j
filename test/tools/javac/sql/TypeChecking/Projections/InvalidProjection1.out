error: The determined sql query type java.lang.Double cannot be assigned to the expected type int
1 error